packages/maskbook/src/plugins/RedPacket/SNSAdaptor/RedPacketPast.tsx

```diff
@@ -0,0 +1,110 @@
+import { makeStyles } from '@masknet/theme'
+import { useCallback, useState } from 'react'
+import AbstractTab, { AbstractTabProps } from '../../../components/shared/AbstractTab'
+import { useI18N } from '../../../utils'
+import { IconURLs } from './IconURL'
+import { RedPacketHistoryList } from './RedPacketHistoryList'
+import { NftRedPacketHistoryList } from './NftRedPacketHistoryList'
+import type { NftRedPacketHistory, NftRedPacketJSONPayload, RedPacketJSONPayload } from '../types'
+import { RedPacketNftMetaKey } from '../constants'
+import { useCompositionContext } from '../../../components/CompositionDialog/CompositionContext'
+import { useCurrentIdentity } from '../../../components/DataSource/useActivatedUI'
+
+enum RpTypeTabs {
+    ERC20 = 0,
```

Comment:
Avoid using `0` as enum value.

Reply:
It's for the tab, then the value should be number